Diagnostics/start-windows-build.ps1

```diff
@@ -0,0 +1,405 @@
+Param(
+    [Parameter(Mandatory=$false)]
+    [string]$GitURL="https://github.com/dcos/dcos-diagnostics", 
+    [Parameter(Mandatory=$false)]
+    [string]$ReviewID,
```

Comment:
I don't think that we need this parameter anymore. This is used for the Mesos reviewboard review request id.